src/main/java/org/ow2/proactive/model/CatalogObject.java

```diff
@@ -32,34 +32,34 @@
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
-import lombok.Setter;
 
 
+/**
+ * This is catalog object model
+ */
 @Getter
 @RequiredArgsConstructor
 @AllArgsConstructor
 public class CatalogObject {
 
-    private final int bucketID;
+    private final int bucket_id;
 
     private final String name;
 
     private final String kind;
 
-    private final String contentType;
+    private final String content_type;
 
-    private final String commitMessage;
+    private final String commit_message;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "commit_message" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00116)